cmd/buildah/list.go

```diff
@@ -10,7 +10,7 @@ import (
 var (
 	listFlags = []cli.Flag{
 		cli.BoolFlag{
-			Name:  "quiet",
+			Name:  "quiet, q",
```

Comment:
Thought we were doing 'omit' here to kill the column heading and 'quiet' to show just the id column.  fwiw, docker ps --quiet caused it to display on the numeric ids.